src/ralph/api/auth.py

```diff
@@ -108,6 +109,13 @@ def get_stored_credentials(auth_file: Path) -> ServerUsersCredentials:
     return ServerUsersCredentials.parse_file(auth_file)
 
 
+@cached(
+    TTLCache(maxsize=10, ttl=3600),
```

Comment:
Also, a default `maxsize` of `10` seems low to me: if I understand correctly, it means by default that we allow to cache the `10` latest logged-in users. WDYT?

Reply:
On the opposite, a TTL of one hour seems perfectly fine to me.

Reply:
Switched to a `maxsize` of `100`, now configurable